docs/docs/en/guide/task/emr.md

```diff
@@ -19,11 +23,13 @@ Amazon EMR task type, for creating EMR clusters on AWS and running computing tas
 | Task priority | When the number of worker threads is insufficient, execute in the order of priority from high to low, and tasks with the same priority will execute in a first-in first-out order. |
 | Worker grouping | Assign tasks to the machines of the worker group to execute. If `Default` is selected, randomly select a worker machine for execution. |
 | Times of failed retry attempts | The number of times the task failed to resubmit. You can select from drop-down or fill-in a number. |
-| Failed retry interval: The time interval for resubmitting the task after a failed task. You can select from drop-down or fill-in a number. |
+| Failed retry interval | The time interval for resubmitting the task after a failed task. You can select from drop-down or fill-in a number. |
 | Timeout alarm | Check the timeout alarm and timeout failure. When the task runs exceed the "timeout", an alarm email will send and the task execution will fail. |
-| JSON | JSON corresponding to the [RunJobFlowRequest](https://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/services/elasticmapreduce/model/RunJobFlowRequest.html) object, for details refer to [API_RunJobFlow_Examples](https://docs.aws.amazon.com/emr/latest/APIReference/API_RunJobFlow.html#API_RunJobFlow_Examples). |
+| Program Type | Select the program type. If it is `RUN_JOB_FLOW`, you need to fill in `jobFlowDefineJson`, if it is `ADD_JOB_FLOW_STEPS`, you need to fill in `stepsDefineJson`. |
+| jobFlowDefineJson | JSON corresponding to the [RunJobFlowRequest](https://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/services/elasticmapreduce/model/RunJobFlowRequest.html) object, for details refer to [API_RunJobFlow_Examples](https://docs.aws.amazon.com/emr/latest/APIReference/API_RunJobFlow.html#API_RunJobFlow_Examples). |
+| stepsDefineJson | JSON corresponding to the [AddJobFlowStepsRequest](https://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/services/elasticmapreduce/model/AddJobFlowStepsRequest.html) object, for details refer to [API_AddJobFlowSteps_Examples](https://docs.aws.amazon.com/emr/latest/APIReference/API_AddJobFlowSteps.html#API_AddJobFlowSteps_Examples). |
```

Comment:
These two parameters are not two types, but two elements that need to be filled in the form. I think they need to be specified separately

Reply:
> But does we can fill both two element in the single task?

 No, must to have a type to tell the difference, because the API that connects to AWS is completely different

Reply:
> BTW, can you add some screenshots for this task? I find it missing the screenshot in our docs, you can find some example in  https://dolphinscheduler.apache.org/en-us/docs/dev/user_doc/guide/task/shell.html

yes, i can

Reply:
Done !

Reply:
> > But does we can fill both two element in the single task?
> 
> No, must to have a type to tell the difference, because the API that connects to AWS is completely different

If they can not fill the form in the same time, I think they should be in two different way here. but I do not have strong opinion